README.md

```diff
@@ -1219,11 +51,11 @@ As an open source project, Ballerina welcomes contributions from the community.
 
 For more information, go to the [contribution guidelines](https://github.com/ballerina-platform/ballerina-lang/blob/master/eventhub/CONTRIBUTING.md).
 
-## Code of Conduct
+## Code of conduct
 
 All the contributors are encouraged to read the [Ballerina Code of Conduct](https://ballerina.io/code-of-conduct).
```

Comment:
Please use the same sentence as in the guideline

Reply:
Resolved and amended the changes. Thank you for the suggestion.